inc/features/blocks/namespace.php

```diff
@@ -350,12 +399,14 @@ function get_views( string $block_id, ?int $post_id = null ) {
 	$result = Utils\query( $query );
 
 	if ( ! $result ) {
-		return [
+		$data = [
 			'loads' => 0,
 			'views' => 0,
 			'conversions' => 0,
 			'audiences' => [],
 		];
+		wp_cache_set( $key, $data, 'altis-xbs', MINUTE_IN_SECONDS );
```

Comment:
Does `Utils\query()` return this cached value?

Reply:
No, it'll return null on failure but log an error - we still want to cache something though to avoid thrashing, but with a short expiry.